Testscripts/Windows/VERIFY-VF-HOT-REMOVE-XDP.ps1

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM, build and Install xdpdump application
+    Verify VF Hot removal and add works as expected
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+# This function will add or remove VF from VM
+function Set-VF {

```

Comment:
Please check out other bash function name convention inside. I think we used 'underscored line' or 'no sumbol'

Reply:
fixed.